beamer/tests/health/test_notify.py

```diff
@@ -0,0 +1,32 @@
+from beamer.health.notify import NotificationState
+
+
+def test_value_is_set(tmpdir):
+    state = NotificationState(tmpdir)
+
+    state.update("a", "RequestExpired")
+
+    assert state.is_set("a", "RequestExpired") is True
+    assert state.is_set("b", "RequestExpired") is False
+
+
+def test_value_is_set_on_already_existing_breadcrumb(tmpdir):
+    state = NotificationState(tmpdir)
+
+    state.update("a", "RequestExpired")
+    state.update("a", "UnclaimedFill")
+
+    assert state.is_set("a", "RequestExpired") is True
+    assert state.is_set("a", "UnclaimedFill") is True
```

Comment:
Same here, as above. Strange that none of our linting tools are complaining about it.